docs/ref/settings.txt

```diff
@@ -3240,6 +3249,16 @@ You may need to :ref:`configure these files to be served in development
 <serving-static-files-in-development>` and will definitely need to do so
 :doc:`in production </howto/static-files/deployment>`.
 
+.. note::
+
+    If :setting:`STATIC_URL` is a path (i.e., not a URL) that does *not* begin
```

Comment:
Even if  `STATIC_URL` begins with a slash, `SCRIPT_NAME` still should be appended IMO, because Django app should work without code changed both locally (no SCRIPT_NAME, no subfolder) and in prod (in subfolder)